### ISSUE TITLE
[Question] Edit ThingsBoard login page HTML code.

### ISSUE THREAD

**Issue by georgemoutsios** [thingsboard/thingsboard#6992]
Hi, I want to know if it is possible to edit the HTML code for ThingsBoard's login page (not Whitelabeling or just css). Basically I want to run a custom script when the page loads.

**Environment**
 * OS:  Ubuntu 20.04.3 LTS
 * ThingsBoard: 3.2.2 PE
 * Browser: Google Chrome

**Comment by skewty** [comment]
Yes. Extract contents of thingsboard jar file. Make the changes to the source code and create the jar file again.
If you are looking for a GUI to change every bit of the app, it doesn't exist.

The ThingsBoard client side code is mostly Angular version 11 components.

**Comment by Ultrazombie** [comment]
We inform you that your GitHub ticket has been inactive for some time, and we will close it. If you have any more concerns or encounter similar issues, please don't hesitate to contact us.
